HaxeManual/10-std.tex

```diff
@@ -400,7 +400,7 @@ \section{Lambda}
 }
 \end{lstlisting} 
 
-This example demonstrates the Lambda count, has, foreach and fold function on a set of ints.
+This example demonstrates the Lambda count, has, for each and fold function on a set of ints.
```

Comment:
No, `foreach` is the name of the function. If anything, they should be in `\expr`.